src/main/java/seedu/address/model/Model.java

```diff
@@ -53,22 +54,32 @@
     ReadOnlyAddressBook getAddressBook();
 
     /**
-     * Returns true if a client with the same identity as {@code client} exists in the address book.
+     * Returns true if a client with the same identity as {@code Client} exists in the address book.
      */
     boolean hasClient(Client client);
 
     /**
      * Deletes the given client.
      * The client must exist in the address book.
      */
-    void deleteClient(Client target);
+    void deleteClient(Client client);
 
     /**
      * Adds the given client.
-     * {@code client} must not already exist in the address book.
+     * {@code Client} must not already exist in the address book.
      */
     void addClient(Client client);
 
+    /**
+     * Delete the given Procedure.
+     */
+    void deleteProcedure(Procedure procedure);
+
+    /**
+     * Delete the given Procedure.
```

Comment:
Just a little typo here :) Delete => Add

Reply:
Thank you! That was very meticulous of you! :)